linear_search/Python/linear_search.py

```diff
@@ -0,0 +1,11 @@
+def linear_search(values, search_for):
+    search_at = 0
+    search_res = False
+    while search_at < len(values) and search_res is False:
+        if values[search_at] == search_for:
+            search_res = True
+        else:
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpejamxp3k/linear_search/Python/linear_search.py
+++ b/tmp/tmpejamxp3k/linear_search/Python/linear_search.py
@@ -8,4 +8,3 @@
             search_at = search_at + 1
 
     return search_res
-    ```